src/test/data/merkle_roots.json

```diff
@@ -0,0 +1,18 @@
+[
+	"caea625775792fb7553c4e21915b1713ae18b9d03d8e04786014dac6fd83bcd7",
+	"e2ef3d63416cec2325a7468cbc1eefc91575105620c7b76312fc3ce04d70be67",
+	"32dd56bb1128fba775a0dbd12a15e12378ade73d67e04c57ee6336e5d2a78530",
+	"432f637c762e1fe0d17d81d1e4e99c929ddcc7472d13c250897c9b693bd8d360",
+	"cc73383f18247923e9472ac7aa97851a9740d731956683eec67c6dff00427b47",
+	"3197598c9f2345402ac4e5b80ff7eda14e22a7fb954270846ceebb403c7f5c94",
+	"2d97b4feb3a45b79206315da4cf625d3ee739794344f51186b39d9d1c20b9526",
+	"ac42a2e51c799ee492f2ec961780ce87964e7baecea47b8ee77519140c3dd364",
+	"cd1a6cdc8a1db28aedcbd96bfbdcfde5429db68e1d38c4cfb5d0c91bc195ee0e",
+	"d20bb06d178f8a514620c7aa549c8467912abdd97c5e892ef5715ad9ed116c7b",
+	"f6da8716682d600f74fc16bd0187faad6a26b4aa4c24d5c055b216d94516847e",
+	"54c1402e1dfc649dd38d23230ff1cb786f49c85c00e92b378c7b5f3312c5c628",
+	"976d045f7f84687ff64b95f5a686d943b6f2f4f2e4159d552f12ed7c907b65c8",
+	"4480b49acaff4f737d156f7c8dcbbd035f0c4a8450d70eed3c76d30f69d71ece",
+	"a867397884b157b1db4dd03dc2fd6c40c4650e70b61441d4325e0b18d5280fb6",
+	"52c8456d3538eed3b73778c596c1993ac6d6c337ae5e338391ce6cae58296dec"
+]
```

Comment:
I like having newlines at the end of files so that when you `cat` them your terminal prompt doesn't get pushed over by the last line of the file, but that's just a nitpick.